bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -247,78 +250,7 @@ public void configurationUpdated(Thing thing) {
         @Override
         public void changeThingType(final Thing thing, final ThingTypeUID thingTypeUID,
```

Comment:
if we decide to rename changeThingType to migrateThingType for ThingTypeMigrator then we should think about to rename this operation as well